enderio-base/src/main/java/com/enderio/base/client/renderer/teleportation/RenderTeleportTargets.java

```diff
@@ -0,0 +1,34 @@
+package com.enderio.base.client.renderer.teleportation;
+
+import com.enderio.base.common.handler.travel.ITravelTarget;
+import com.enderio.base.common.handler.travel.TravelRegistry;
+import com.enderio.base.common.handler.travel.TravelSavedData;
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.minecraft.client.Camera;
+import net.minecraft.client.Minecraft;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.RenderLevelLastEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(value = Dist.CLIENT)
+public class RenderTeleportTargets {
+
+    @SubscribeEvent
+    public static void renderLevel(RenderLevelLastEvent event) {
+        TravelSavedData data = TravelSavedData.getTravelData(Minecraft.getInstance().level);
+        for (ITravelTarget target : data.getTravelTargets()) {
+            PoseStack poseStack = event.getPoseStack();
+            poseStack.pushPose();
+            Camera mainCamera = Minecraft.getInstance().gameRenderer.getMainCamera();
+            Vec3 projectedView = mainCamera.getPosition();
+            poseStack.translate(-projectedView.x, -projectedView.y, -projectedView.z);
+
+            //TODO: Scale, View Bobbing, (FOV/Creative Flight offset stuff)
```

Comment:
it wasn't as they use a BlockEntityRenderer for some things, which wouldn't work for us, because we have the big teleporter with infinite range. Also I wanted to get the logic done first, so the Draft would be cleaner